### PR TITLE
Add notes about using hera as a shared library

### PR DESCRIPTION
**Pull request by jwasinger**
fixes https://github.com/ewasm/testnet/issues/54

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +23,19 @@ The config is in [ewasm-testnet-cpp-config.json](ewasm-testnet-cpp-config.json).
 
 Example node with mining on single CPU core, with no bootstrap:
 
+1) Build (hera)[https://github.com/ewasm/hera] as a shared library:
+```
+git clone https://github.com/ewasm/hera --recursive
+cd hera
+cmake -DBUILD_SHARED_LIBS=ON .
```
Comment (axic) [inline README.md]: Two dots...

Comment (axic) [reply README.md]: But I would suggest to copy it from the geth section:
```
 	mkdir build && cd build
 	cmake .. -DBUILD_SHARED_LIBS=ON
 	cmake --build .
```

Comment (jwasinger) [reply README.md]: Fixed now.

Comment (jwasinger) [reply README.md]: Done

**Pull request merged**